src/test/cli/rbd/help.t

```diff
@@ -1102,6 +1103,8 @@
     --read-only           mount read-only
     --exclusive           forbid other clients write
     --device arg          specify nbd device
+    --nbds_max arg        Override for module param nbds_max
+    --max_part arg        Override for module param max_part
   
```

Comment:
The description for other options starts in lower case, please, be consistent.

Also it looks it would look more consistent without "for": "override module param nbds_max"...